examples/StandardFirmataBLE/bleConfig.h

```diff
@@ -106,6 +106,15 @@ BLEStream stream;
 // BLEStream stream(BLE_REQ, BLE_RDY, BLE_RST);
 // #endif
 
+/*
+ * Adafruit Bluefruit nRF52 Boards
+ */
+#ifdef ARDUINO_NRF52_ADAFRUIT
+#include "utility/BluefruitStream.h"
+BLEStream stream;
+#endif
```

Comment:
I see I overlooked the way you are using ANALOG_TO_PIN so my suggestion here would not work. I could try to resolve this on the Firmata side by allowing either a channel or pin number but it may be a while before I have time to work on a fix.